Assets/SteamVR_Unity_Toolkit/Scripts/Helper/DeviceFinder.cs

```diff
@@ -35,9 +35,10 @@ public static SteamVR_TrackedObject TrackedObjectByIndex(uint controllerIndex)
         public static Transform HeadsetTransform()
         {
 #if (UNITY_5_4_OR_NEWER)
-        return GameObject.FindObjectOfType<SteamVR_Camera>().GetComponent<Transform>();
-#endif
+            return GameObject.FindObjectOfType<SteamVR_Camera>().GetComponent<Transform>();
+#else
```

Comment:
It does do it in a slightly different way, but it stops the 'unreachable code' warning that comes up each time the code compiles while using Unity 5.4+. (Notice that without the #else, the second line will still be in the compiled code, but be unreachable.

I can keep the change locally. I just thought anyone else using the beta might benefit from it as well.